src/blurredLocation.js

```diff
@@ -70,7 +71,7 @@ BlurredLocation = function BlurredLocation(options) {
   function geocodeStringAndPan(string, onComplete) {
     var url = "https://maps.googleapis.com/maps/api/geocode/json?address="+string.split(" ").join("+");
     var Blurred = $.ajax({
-        async: false,
+        async: options.map = options.map || new L.Map(options.mapID, {}).setView([options.location.lat, options.location.lon], options.zoom);,
```

Comment:
Oh interesting! What does this line do?

Reply:
Hi @jywarren, thanks for pointing it out! Should be fixed now. :)